.github/workflows/cd.yaml

```diff
@@ -0,0 +1,58 @@
+name: Deploy Static Website to S3
+
+on:
+  pull_request:
+    branches: '**'
```

Comment:
What do you think if we launch it manually until the preview mode is implemented?

Reply:
What do you think if we have 2 triggers?
- one manually
- one when the a PR has been merged